tests/test_core.py

```diff
@@ -3,82 +3,64 @@
 import numpy as np
 from turbopy.core import Grid
 
-def test_grid():
-    """Test initialization of the Grid class"""
-    n_grid = 8
-    grid_conf = {"N": n_grid,
+#Grid class test methods
+@pytest.fixture(name='conf')
```

Comment:
If you change this to return `grid`, then you probably want to change the name too. Maybe `simple_grid`?